### PR TITLE
Jhalackova bsc#1223771

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Adding how to re-encrypt raw images.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

File: tasks/deployment-reencryption.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  </warning>
+  <note>
+    <title>The step is not needed if you deployed your system using &jeosfirstboot;</title>
+    <para>
+    &jeosfirstboot; asks for a new passphrase during the deployment phase. After you enter it, the
+    system is re-encrypted automatically, thus no further action is needed.
+    </para>
+  </note>
+  <para>
+    &productname; encrypted images ar delivered with a default LUKS passphrase. To secure your
```
Comment (lcp) [inline tasks/deployment-reencryption.xml:46]: typo: 'ar' -> 'are'

File: tasks/deployment-reencryption.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    system, make sure that you change it after the system is deployed. To do so, proceed as
+    describe bellow. Make sure to perform the steps in the same shell session.
+  </para>
+  
+  <procedure>
+    <step>
+      <para>
+        Import the needed functions to your shell:
+      </para>
+      <screen>&prompt.root;source /shr/share/fde/luks</screen>
```
Comment (lcp) [inline tasks/deployment-reencryption.xml:56]: typo: /shr/share/fde/luks -> /usr/share/fde/luks

File: tasks/deployment-reencryption.xml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      <para>
+       The topic gives instructions on how to change the default passphrase after deploying the encrypted image.
+      </para>
+    </abstract>
+  </info>
+  <warning>
+    <title>The system is not secured</title>
+    <para>
+      The system is not secured, therefore, do not store any sensitive data in it, until the disk
+      reencryption is complete.
```
Comment (dariavladykina) [inline tasks/deployment-reencryption.xml:35]:
```suggestion
      The system is not secured. Thus, do not store any sensitive data in it until the disk
      reencryption is complete.
```

File: tasks/deployment-reencryption.xml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    </para>
+  </warning>
+  <note>
+    <title>The step is not needed if you deployed your system using &jeosfirstboot;</title>
+    <para>
+    &jeosfirstboot; asks for a new passphrase during the deployment phase. After you enter it, the
```
Comment (dariavladykina) [inline tasks/deployment-reencryption.xml:41]:
```suggestion
    &jeosfirstboot; prompts for a new passphrase during the deployment phase. After you enter it, the
```

File: tasks/deployment-reencryption.xml [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    system is re-encrypted automatically, thus no further action is needed.
```
Comment (dariavladykina) [inline tasks/deployment-reencryption.xml:42]:
```suggestion
    system is reencrypted automatically, thus no further action is needed.
```

File: tasks/deployment-reencryption.xml [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    </para>
+  </note>
+  <para>
+    &productname; encrypted images are delivered with a default LUKS passphrase. To secure your
+    system, make sure that you change it after the system is deployed. To do so, proceed as
+    describe bellow. Make sure to perform the steps in the same shell session.
```
Comment (dariavladykina) [inline tasks/deployment-reencryption.xml:48]:
```suggestion
    system, make sure that you change it after the system is deployed. To do so, proceed as
    described below. Perform the steps in the same shell session.

File: tasks/deployment-reencryption.xml [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+      <screen>&prompt.root;luks_name=$(expr "`df --output=source / | grep /dev/`" :
+      ".*/\(.*\)")</screen>
+      <para>
+        and:
+      </para>
+      <screen>&prompt.root;luks_dev=$(luks_get_underlying_device "$luks_name")</screen>
+    </step>
+    <step xml:id="creating-key-files">
+      <para>
+        Create a key file that stores the default passphrase '1234' and a key file with the new passphrase.
```
Comment (dariavladykina) [inline tasks/deployment-reencryption.xml:71]:
```suggestion
        Create a key file that stores the default passphrase <emphasis>1234</emphasis> 
        and a key file with the new passphrase.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions from me. Thanks!

**Pull request merged**